bin/magpie-postgres-daemon.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#############################################################################
+#  Copyright (C) 2013-2015 Lawrence Livermore National Security, LLC.
+#  Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
+#  Written by Albert Chu <chu11@llnl.gov>
+#  LLNL-CODE-644248
+#  
+#  This file is part of Magpie, scripts for running Hadoop on
+#  traditional HPC systems.  For details, see https://github.com/llnl/magpie.
+#  
+#  Magpie is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  
+#  Magpie is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with Magpie.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+
+# This script launches Postgres for the user
+if [ "$1X" == "X" ]
+then
+    echo "User must specify start or stop as first argument"
+    exit 1
+fi
+
+if [ "$1" != "start" ] && [ "$1" != "stop" ]
+then
+    echo "User must specify start or stop as first argument"
+    exit 1
+fi
+
+if [ "$2X" == "X" ]
+then
+    echo "User must specify Hive configuration path as second argument"
+    exit 1
+fi
+
+if [ ! -d $2 ]
+then
+    echo "Hive configuration path is not a directory"
+    exit 1
+fi
+
+# source the hive-env.sh file
+source $2/hive-env.sh
+myhostname=`hostname`
+
+if [ "$1" == "start" ]
+then
+  if [ ! -d $PGDATA ]
+  then
+    echo "${PGDATA} not found!  Initializing under the assumption that this is a new instance"
+    mkdir -p $PGDATA
```

Comment:
should check for error on mkdir and exit if needed

Reply:
Added